examples/stm32/nucleo-h723zg-make-baremetal-builtin/hal.h

```diff
@@ -8,6 +8,16 @@
 
 #pragma once
 
+#define BTN PIN('C', 13)   // User push button
+#define LED1 PIN('B', 0)   // On-board LED pin (green)
+#define LED2 PIN('E', 1)   // On-board LED pin (yellow)
+#define LED3 PIN('B', 14)  // On-board LED pin (red)
+#define LED LED2           // Use yellow LED for blinking
+
+#ifndef UART_DEBUG
+#define UART_DEBUG USART3
+#endif
+
```

Comment:
I like defines after includes, as it was before... just a matter of taste and having all necessary stuff at top
Plus, all hal.h have UART_DEBUG near the UART code